GNUmakefile

```diff
@@ -77,10 +82,31 @@ teamcity-servicemap-generate:
 	cd tools/teamcity-generator;\
 		go run . --output $(OUTPUT_PATH) --version $(VERSION)
 
+
+clean-provider:
+	cd $(OUTPUT_PATH);\
+		go mod download;\
+		find . -type f -not -wholename "./.git*" -not -wholename "./.changelog*" -not -name ".travis.yml" -not -name ".golangci.yml" -not -name "CHANGELOG.md" -not -name "CHANGELOG_v*.md" -not -name "GNUmakefile" -not -name "docscheck.sh" -not -name "LICENSE" -not -name "README.md" -not -wholename "./examples*" -not -name ".go-version" -not -name ".hashibot.hcl" -print0 | xargs -0 git rm > /dev/null
+
+clean-tgc:
+	cd $(OUTPUT_PATH);\
+		rm -rf ./tfplan2cai/testdata/templates/;\
+		rm -rf ./tfplan2cai/testdata/generatedconvert/;\
+		rm -rf ./tfplan2cai/converters/google/provider;\
+		rm -rf ./tfplan2cai/converters/google/resources;\
+		rm -rf ./cai2hcl/*;\
+		find ./tfplan2cai/test/** -type f -exec git rm {} \; > /dev/null;\
+
 tgc:
 	cd mmv1;\
 		bundle;\
-		bundle exec compiler -e terraform -f tgc -v beta -o $(OUTPUT_PATH) $(mmv1_compile);\
+		bundle exec compiler -e terraform -f tgc -v beta -o $(OUTPUT_PATH)/tfplan2cai $(mmv1_compile);\
+		bundle exec compiler -a -e terraform -f tgc_cai2hcl -o $(OUTPUT_PATH)/cai2hcl -v beta;\
```

Comment:
cai2hcl should be taking in `$(mmv1_compile)` as well so that the product list is consistent. Could the order of args in this line & the previous line match so that they're easier to compare?